code/datums/bodypart_overlays/mutant_bodypart_overlay.dm

```diff
@@ -59,7 +59,10 @@
 
 	return appearance
 
-/datum/bodypart_overlay/mutant/color_image(image/overlay, obj/item/bodypart/limb)
+/datum/bodypart_overlay/mutant/color_image(image/overlay, layer, obj/item/bodypart/limb)
+	if(!sprite_datum || !overlay)
```

Comment:
When can these be null? If there’s no overlay or sprite_datum something is very wrong

Reply:
They can be null when the proc just above returns a `null` if there's no `sprite_datum`. I figured I'd add the same kind of handling here, otherwise it'll just runtime when the one above handles it.

Reply:
I think it should runtime then, sprite_datum shouldn’t be null (IIRC) so if it is then at least it’ll alert us somethings broken? Another maintainer might disagree though

Reply:
Yeah, although I feel like it's probably the other proc that should runtime, and most likely have a `CRASH()` rather than just a runtime.

Reply:
Didnt mean to hit resolve. Yes you’re definitely right, that’s way better fir debugging

Reply:
You can remove the check here though now that we check in get_image